src/sider/sykmeldinger/sykmelding/sykmeldingOpplysninger/SykmeldingSeksjon.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactNode } from "react";
+import { Heading } from "@navikt/ds-react";
+
+interface SykmeldingSeksjonProps {
+  tittel: string;
+  children?: ReactNode;
```

Comment:
Det skal kanskje heller ikke gå an å lage en sykmeldingseksjon uten innhold? 😊
```suggestion
  children: ReactNode;
```